sdk/core/core-amqp/src/util/constants.ts

```diff
@@ -77,6 +77,8 @@ export const defaultMaxRetries = 3;
 export const defaultMaxRetriesForConnection = 150;
 export const defaultDelayBetweenOperationRetriesInSeconds = 5;
 export const defaultDelayBetweenRetriesInSeconds = 15;
+export const defaultMaxDelayForExponentialRetryInMs = 50000;
+export const defaultMinDelayForExponentialRetryInMs = 1;
```

Comment:
How did we arrive at these defaults? Assuming that we wan't to mirror the retry policy in core-http, these should be different

Reply:
Oh, the defaults seemed different in general across these 2, such as for delay between retries to be same as well (5 vs 15 vs 30 seconds based on type of retry).
Fixed this as well to use default as 30 sec.